css/properties/white-space.json

```
@@ -96,6 +96,54 @@
             }
           }
         },
+        "nowrap": {
+          "__compat": {
+            "description": "<code>nowrap</code>",
+            "support": {
+              "chrome": {
+                "version_added": "1"
+              },
+              "chrome_android": {
+                "version_added": "18"
+              },
+              "edge": {
+                "version_added": "12"
+              },
+              "firefox": {
+                "version_added": "3.5"
+              },
+              "firefox_android": {
+                "version_added": "4"
+              },
+              "ie": {
+                "version_added": "8"
+              },
+              "opera": {
+                "version_added": "9.5"
+              },
+              "opera_android": {
+                "version_added": "46"
+              },
+              "safari": {
+                "version_added": "3"
```

Safari's first release is also sometime after this property was standardized. All of the WebKit bugs I could find for `nowrap` were about bugs in the implementation, but none of them seemed to indicate the feature was in any way new. Seems safe to set this as the first release as well:

```suggestion
                "version_added": "1"
```